### PR TITLE
Added support for add-ons to have Sensor Glasses Armor Items

### PR DESCRIPTION
**Pull request by MJRLegends**
Hello,

So please let me explain why im making this change. 

In my addon i have Pressure and Radiation System that requires the player to wear a space suit at all times. So im working on a module system for players to install different features to there space suit so players are not missing out on features that require use of armor slots like the sensor glasses.

Making this change i've done allows any armour anyone makes be able to work as sensor glasses as well as the Sensor Glasses item.

Basically i really need this :D 

I've tested the Sensor Glasses item works fine still with the added ability of allowing my feature in my addon to work too :D

### CODE REVIEW

**Comment by radfast** [comment]
Seems OK, we can merge this - it's a good idea.

When making commits in future, please don't re-order the imports (if your IDE has a feature which automatically does that on save, switch it off).

Re-ordering imports makes the process of merging into upstream versions take longer, because `Git merge` does not handle Java imports very well.
